src/common/components/form/fields/input/NumberInputField.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { KeyboardEvent } from 'react';
+import { FieldProps, getIn } from 'formik';
+import classnames from 'classnames';
+import { useTranslation } from 'react-i18next';
+
+import Input from '../../../input/Input';
+import styles from './inputField.module.scss';
+
+interface NumberInputField extends FieldProps {
+  id: string;
+  label: string;
+  maxLength: number;
+}
+
+const NumberInputField: React.ComponentType<NumberInputField> = ({
```

Comment:
I dont want to put more logic to single InputField component so i make new component which look almost same for number input only. Sorry for duplication